### PR TITLE
Fix delete memory usage

### PR DESCRIPTION
**Pull request by MikhailBurdukov**
Do not keep full metadata for all backups in the same time.

### CODE REVIEW

File: ch_backup/config.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def _as_seconds(t: str) -> int:
     },
     "loguru": {
         "formatters": {
-            "ch-backup": "{time:YYYY-MM-DD H:m:s,SSS} {process.name:11} {process.id:5} [{level:8}] {extra[logger_name]}: {message}",
+            "ch-backup": "{time:YYYY-MM-DD HH:mm:ss,SSS} {process.name:11} {process.id:5} [{level:8}] {extra[logger_name]}: {message}",
```
Comment (Alex-Burmak) [inline ch_backup/config.py:157]: Looks unrelated to the memory usage fix.

Comment (aalexfvk) [reply ch_backup/config.py:157]: It looks like the branch was forked not from main, because this changes already in main.

File: ch_backup/ch_backup.py [hunk 2]
```diff
@@ -309,7 +309,7 @@ def delete(
         retained_backups = []
         with self._context.locker():
             for i, backup in enumerate(
-                self._context.backup_layout.get_backups(use_light_meta=False)
+                self._context.backup_layout.get_backups(use_light_meta=True)
```
Comment (Alex-Burmak) [inline ch_backup/ch_backup.py:312]: Suggest to add a comment describing why it's important to use light metadata here. Something like `# Use light metadata in backups iteration to avoid high memory usage.`.

**Comment by MikhailBurdukov** [comment]
Fixed review suggestions and made same changes for  `purge` command.

**Review by Alex-Burmak — APPROVED** [review]

**Review by aalexfvk — APPROVED** [review]
